server/spec/one_sub_pool_per_stack_spec.rb

```diff
@@ -342,6 +342,9 @@
     # Simulate migration
     @host2_client.update_consumer({:guestIds => [{'guestId' => @guest_uuid}]})
 
+    # Need to wait a moment here for MySQL to catch up
+    sleep 2
```

Comment:
This seems error prone if the system is under a lot of load. Is there an alternative method to watch for completion of the update_consumer?